test_after_commit.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new name, TestAfterCommit::VERSION do |s|
   s.license = 'MIT'
 
   s.required_ruby_version = '>= 2.0.0'
-  s.add_runtime_dependency "activerecord", [">= 3.2", "< 5.0"]
+  s.add_runtime_dependency "activerecord", ">= 3.2"
```

Comment:
this should be `< 7.1` and add the missing gemfiles
(can remove all the old 4.x gemfiles)
and update the readme to say that this makes sense for non-rails only

Reply:
Will do